### ISSUE TITLE
Question about the cascaded encoder-decoder

### ISSUE THREAD

**Issue by Kittywyk** [open-mmlab/mmsegmentation#1075]
Hello!
When I checked the implementation of cascaded encoder-decoder like ocrnet_r50-d8, what I understood is that the output logit of the first decode head is fed into the second decode head. But why not feed the feature maps before classifying each pixel? Did I misunderstand code?
Thanks in advance for your help.

**Comment by MengzhangLI** [comment]
Hi, sorry for late reply.

`But why not feed the feature maps before classifying each pixel`

Because classifying each pixel of first decoder head is necessary, actually two serial decoder head both need get logits of feature map. Your understanding is correct.

**Comment by Kittywyk** [comment]
Got it. Thank you.

发自我的iPhone


------------------ Original ------------------
From: MengzhangLI ***@***.***&gt;
Date: Thu, Dec 16, 2021 5:40 PM
To: open-mmlab/mmsegmentation ***@***.***&gt;
Cc: Kittywyk ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [open-mmlab/mmsegmentation] Question about the cascadedencoder-decoder (Issue #1075)
